scripts/exrrfs_run_maxrh.ksh

```diff
@@ -0,0 +1,286 @@
+#!/bin/bash
+
+#
+#-----------------------------------------------------------------------
+#
+# Source the variable definitions file and the bash utility functions.
+#
+#-----------------------------------------------------------------------
+#
+. ${GLOBAL_VAR_DEFNS_FP}
+. $USHdir/source_util_funcs.sh
+#
+#-----------------------------------------------------------------------
+#
+# Save current shell options (in a global array).  Then set new options
+# for this script/function.
+# 
+#-----------------------------------------------------------------------
+#   
+{ save_shell_opts; set -u -x; } > /dev/null 2>&1
+#   
+#-----------------------------------------------------------------------
+#
+# Get the full path to the file in which this script/function is located 
+# (scrfunc_fp), the name of that file (scrfunc_fn), and the directory in
+# which the file is located (scrfunc_dir).
+#
+#-----------------------------------------------------------------------
+#
+scrfunc_fp=$( readlink -f "${BASH_SOURCE[0]}" )
+scrfunc_fn=$( basename "${scrfunc_fp}" )
+scrfunc_dir=$( dirname "${scrfunc_fp}" )
+#
+#-----------------------------------------------------------------------
+#
+# Print message indicating entry into script.
+#
+#-----------------------------------------------------------------------
+#
+print_info_msg "
+========================================================================
+Entering script:  \"${scrfunc_fn}\"
+In directory:     \"${scrfunc_dir}\"
+
+This is the ex-script for the task that runs the post-processor (UPP) on
+the output files corresponding to a specified forecast hour.
+========================================================================"
+case $MACHINE in
+
+  "WCOSS2")
+    module load wgrib2/2.0.8
+    module list
+    wgrib2=/apps/ops/prod/libs/intel/19.1.3.304/wgrib2/2.0.8_wmo/bin/wgrib2
+    ;;
+
+  "HERA")
```

Comment:
This part for loading modules should be removed. For other machines, the module file used for building executables is loaded for every script. You don't have to load the modules in the scripts again.